src/renderer/i18n/en.js

```diff
@@ -211,6 +212,16 @@ const English = {
     gotoUpdate: "Update Firmware",
     reconnect: "Reconnect",
     reconnectDescription: `There's a possibility that we misdetected the capabilities of the keyboard, or that the keyboard was starting up while we connected. In this case, you can try clicking the "{{buttonName}}" button to attempt a reconnect, and look for the necessary features again. Reconnecting is useful if you're sure there was a temporary failure upon previous attempts, and the problem has been resolved.`
+  },
+  systeminfo: {
+    title: "System Information",
+    contents: `Chrysalis can collect a debug bundle you can share with the developers as a troubleshooting aid. The bundle will include information about your operating system and its version; Chrysalis' version; and if there's a connected device, its name, and selected properties of it (such as the keymap and the colormap).`,
```

Comment:
If you're having trouble with Chrysalis, the development team may ask you to send them some debugging information. Chrysalis can help you collect and package up everything you might need to share. This includes error logs, as well as a bit of information about this computer's operating system, connected devices, and Chrysalis itself. Keyboard information will include information about the keyboard's current firmware, as well as key layouts and LED configuration. 

We try hard not to include private information in this bundle, but recommend you read through it before sharing it.